doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing.
+
+However, sometimes the stream can have missing or duplicate records, or send records out of order. For example, if a CDC tool crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
+
+To prevent this kind of error, Materialize has developed the **Open CDC format**, a format for the updates that CDC tools propagate to other systems.  The Open CDC format is designed especially to prevent record duplication or missequencing by including enough information for a downstream data consumer (like Materialize) to recognize when records are missing, duplicated, or out of order. Each record contains:
```

Comment:
Nit, but "this kind of error" could be a few things. The "CDC tool crashes" is the most obvious, as a duplicate entry isn't as clearly an "error". I guess at this point it isn't clear that duplicate entries would lead to an error. Maybe tack on a sentence on the prev paragraph like: "Duplicate entries may cause the reader to think the change happened twice, leading to a state inconsistent with the actual database".

Reply:
In materialize-cdc.md, substantially rephrased.